tbmalt/physics/filling.py

```diff
@@ -27,8 +33,8 @@ def entropy_term(func, eigenvalues: Tensor, fermi_energy: Tensor,
 
 
 def fermi_entropy(eigenvalues: Tensor, fermi_energy: Tensor, kT: float_like,
-                  e_mask: Optional[Union[Tensor, OrbitalInfo]] = None
-                  ) -> Tensor:
+                  e_mask: Optional[Union[Tensor, OrbitalInfo]] = None,
+                  **kwargs) -> Tensor:
```

Comment:
Hey Adam, I am wondering why the kwargs are needed here. Thanks :)

Reply:
At the time I was thinking about allowing for greater flexibility. However, as you suggest, it is likely best to remove it unless it is immediatly necessary. I will remove this.

Reply:
At the time I was thinking about allowing for greater flexibility. However, as you suggest, it is likely best to remove it unless it is immediatly necessary. I will remove this.